lib/Common/Exceptions/Throw.h

```diff
@@ -228,8 +228,4 @@ namespace Js {
     catch (ex) \
     {
 
-#ifdef ENABLE_DEBUG_CONFIG_OPTIONS
-#define RAISE_FATL_INTERNAL_ERROR_IFFAILED(hr) if (hr != S_OK) Js::Throw::FatalInternalError();
-#else
-#define RAISE_FATL_INTERNAL_ERROR_IFFAILED(hr)
-#endif
+#define DEBUGGER_ATTACHDETACH_FATAL_ERROR_IF_FAILED(hr) if (hr != S_OK) Debugger_AttachDetach_fatal_error();
```

Comment:
I would say yes to prevent random issue which can come if this failed. Since this is going in master and will have time to flush any issues.